WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewViewModel.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import Yosemite
+
+/// ViewModel for ThemesPreviewView
+///
+final class ThemesPreviewViewModel: ObservableObject {
+    @Published private(set) var pages: [WordPressPage] = []
```

Comment:
```suggestion
    @Published private(set) var pages: [WordPressPage]
```

We can skip setting an initial value as we are setting this in `init`.